modules/aws-idc-integration/audit-roles/variables.tf

```diff
@@ -0,0 +1,34 @@
+variable "namespace" {
+  description = "Specifies the namespace for the deployment."
+  default     = "common-fate"
+  type        = string
+}
+
+variable "stage" {
+  description = "Defines the stage of the deployment (e.g., 'dev', 'staging', 'prod')."
+  default     = "prod"
+  type        = string
+}
+
+variable "common_fate_aws_account_id" {
+  description = "The ID or the account where Common Fate is deployed"
+  type        = string
+}
+
+variable "assume_role_external_id" {
+  description = "The external id to be used for the IAM policy trust relation"
+  type        = string
+}
+
+
+variable "regions" {
+  description = "The AWS regions to deploy audit roles into."
+  type        = list(string)
+}
```

Comment:
We should switch this to just a `string` - because the IAM roles are global, they should only be deployed to a single region